cxx-qt-lib-headers/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2021 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Be Wilson <be.wilson@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! This crate is a hack so build.rs for cxx-qt-lib and cxx-qt-build both have access to cxx-qt-lib's C++ headers.
```

Comment:
this sounds fun :-)

Reply:
lots of fun :)
![image](https://user-images.githubusercontent.com/9455094/186563157-5d56f9e4-2987-4147-b6b9-e872f1ff4309.png)